### YamlMime:PythonClass
uid: azure.core.exceptions.ClientAuthenticationError
name: ClientAuthenticationError
fullName: azure.core.exceptions.ClientAuthenticationError
module: azure.core.exceptions
inheritances:
- azure.core.exceptions.HttpResponseError
summary: 'An error response with status code 4xx.

  This will not be raised directly by the Azure core pipeline.'
constructor:
  syntax: 'ClientAuthenticationError(message: object | None = None, response: _HttpResponseCommonAPI
    | None = None, **kwargs: Any)'
  parameters:
  - name: message
    defaultValue: None
  - name: response
    defaultValue: None
methods:
- uid: azure.core.exceptions.ClientAuthenticationError.add_note
  name: add_note
  summary: "Exception.add_note(note) \u2013\nadd a note to the exception"
  signature: add_note()
- uid: azure.core.exceptions.ClientAuthenticationError.raise_with_traceback
  name: raise_with_traceback
  summary: 'Raise the exception with the existing traceback.


    *Deprecated since version 1.22.0:* This method is deprecated as we don''t support
    Python 2 anymore. Use raise/from instead.'
  signature: raise_with_traceback() -> None
- uid: azure.core.exceptions.ClientAuthenticationError.with_traceback
  name: with_traceback
  summary: "Exception.with_traceback(tb) \u2013\nset self.__traceback__ to tb and\
    \ return self."
  signature: with_traceback()
attributes:
- uid: azure.core.exceptions.ClientAuthenticationError.args
  name: args
- uid: azure.core.exceptions.ClientAuthenticationError.continuation_token
  name: continuation_token
  signature: 'continuation_token: str | None'
- uid: azure.core.exceptions.ClientAuthenticationError.exc_msg
  name: exc_msg
  signature: 'exc_msg: str'
- uid: azure.core.exceptions.ClientAuthenticationError.exc_traceback
  name: exc_traceback
  signature: 'exc_traceback: TracebackType | None'
- uid: azure.core.exceptions.ClientAuthenticationError.exc_type
  name: exc_type
  signature: 'exc_type: Type[Any] | None'
- uid: azure.core.exceptions.ClientAuthenticationError.exc_value
  name: exc_value
  signature: 'exc_value: BaseException | None'
- uid: azure.core.exceptions.ClientAuthenticationError.inner_exception
  name: inner_exception
  signature: 'inner_exception: BaseException | None'
- uid: azure.core.exceptions.ClientAuthenticationError.message
  name: message
  signature: 'message: str'
